### DISCUSSION TITLE
LAB 02.04 - CARD TRICK Task 2

### DISCUSSION THREAD

**Discussion by KenHerDu** [rramosp/ai4eng.v1#293] [02 - PYTHON]
He realizado la función donde me arroja un resultado correcto pero al calificar no me valida el ultimo punto y me arroja 0,
Unifique los heaps pero nada, igual si solo dejo donde esta la carta elegida también me califica 0 

Que debo realizar en el codigo? que debe salir  al final para que valide ?

![2023-12-19_11h46_47](https://github.com/rramosp/ai4eng.v1/assets/123333303/e9aa0fb2-a0cb-42fd-9aa0-7780fb13eb3f)

@rramosp

**Comment by andr3scontrer4s** [comment]
Hola, me sale el mismo error 
![image](https://github.com/rramosp/ai4eng.v1/assets/164700198/380e9e78-08db-4926-8a4e-bfae9387ea66)

@rramosp como hacemos en este caso?

**Comment by Felipe-RA** [comment]
@andr3scontrer4s 

Compartenos un enlace a tu notebook por acá, y lo miramos
